Python/pathconfig.c

```diff
@@ -272,7 +272,8 @@ Py_SetProgramName(const wchar_t *program_name)
 }
 
 
-wchar_t *
+/* removed in 3.15, but kept for stable ABI compatibility */
+PyAPI_FUNC(wchar_t *)
```

Comment:
Is this the correct to do it @zooba?

Reply:
I'm not 100% sure, we'll find out from the tests :-)

And I don't know exactly what's required for ELF systems to resolve the symbols, or if that even matters. @encukou is a better source of info there. (For Windows, we could also move the implementations into `python3.dll` directly, which would be totally fine, if more complicated. But I think for Linux they need to still be implemented in libpython.)

Reply:
Yes, that's the right way to keep a function in the stable ABI, without exposing it in the public C API.

Reply:
This'll work. You can also put it in an internal header, so tests can get to it normally.